website/src/pages/docs/integrations/integration-with-uwebsockets.mdx

```diff
@@ -93,3 +93,9 @@ App()
     console.log(`Server is running on http://localhost:4000`)
   })
 ```
+
+## Subscriptions with WebSockets
+
+You can also use WebSockets instead of SSE with `graphql-ws`;
+
+[See our full example here](https://github.com/dotansimha/graphql-yoga/blob/main/examples/uwebsockets/src/app.ts)
```

Comment:
Better to have the example in the documentation already and not a redirect. People will come here mostly because of WebSockets.

Reply:
Let me know if it is good now?